[BUG]: site plugin fails

When trying to release 12.0.2 I ran into an issue with the maven-site:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-site-plugin:3.21.0:site (default-site) on project dependency-check-core: Error generating maven-dependency-plugin:3.8.1:analyze-report report: Cannot analyze dependencies: Trying to force use of dependencies which are declared but already detected as used: [ch.qos.logback:logback-classic] -> [Help 1]
```

See the error on the attempted release: https://github.com/jeremylong/DependencyCheck/actions/runs/13030928514/job/36350430369

--- a/core/pom.xml
+++ b/core/pom.xml
@@ -229,6 +229,11 @@ Copyright (c) 2012 Jeremy Long. All Rights Reserved.
             <scope>test</scope>
         </dependency>
         <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-core</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
             <groupId>org.owasp</groupId>
             <artifactId>dependency-check-utils</artifactId>
             <version>${project.parent.version}</version>
@@ -506,9 +511,6 @@ Copyright (c) 2012 Jeremy Long. All Rights Reserved.
                             <version>${maven-dependency-plugin.version}</version>
                             <configuration>
                                 <usedDependencies combine.children="append">
-                                    <!-- logback is our logging implementation during test and is test-scoped due to a lack of a
-                                    test-runtime scope - it should be considered 'used' in the context of dependency:analyze-report -->
-                                    <usedDependency>ch.qos.logback:logback-classic</usedDependency>
                                     <!-- dependencies to be copied for use in unit/integration testcases are, due to
                                     lack of a test-runtime scope, configured as test-scoped / optional and should be
                                     considered used for dependency:analyze-report -->

